mods/voidmulti.sc2mod/base.sc2data/gamedata/validatordata.xml

```diff
@@ -507,6 +508,11 @@
         <Compare value="GT"/>
         <Behavior value="TakenDamage"/>
     </CValidatorUnitCompareBehaviorCount>
+    <CValidatorLocationCompareRange id="ChannelSnipeRange">
+        <WhichLocation Value="TargetUnit"/>
+        <Compare value="LT"/>
+        <Range value="13.5"/>
+    </CValidatorLocationCompareRange>
```

Comment:
Steady Targeting is canceled if the target moves more than 13.5 range away from the ghost while casting (Cast range is 10)

This is the range validator. The patch notes are not accurate as exact 13.5 range will break it already :)